extensions/binding/org.eclipse.smarthome.binding.mqttbroker/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mqttbroker" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="Broker">
```

Comment:
Btw, shouldn't this be a bridge-type instead of a thing-type?

Reply:
https://github.com/eclipse/smarthome/pull/3839#discussion_r135987618